libraries/AP_Airspeed/AP_Airspeed_Health.cpp

```diff
@@ -43,15 +45,36 @@ void AP_Airspeed::check_sensor_ahrs_wind_max_failures(uint8_t i)
         }
         return;
     }
-    const float speed_diff = fabsf(state[i].airspeed-gps.ground_speed());
 
+    // check for airspeed consistent with wind and vehicle velocity using the EKF
+    uint32_t age_ms;
+    float innovation, innovationVariance;
+    if (AP::ahrs().airspeed_health_data(innovation, innovationVariance, age_ms) && age_ms < 1000 && is_positive(innovationVariance)) {
+        state[i].failures.test_ratio = fabsf(innovation) / safe_sqrt(innovationVariance);
+    } else {
+        state[i].failures.test_ratio = 0.0f;
+    }
+    bool data_is_inconsistent;
```

Comment:
```suggestion
    bool data_is_inconsistent = false;
```

Reply:
Unnecessary given the following if structure always sets it.

Reply:
See https://github.com/tridge/ardupilot/commit/69c74dbcc137dda72ac02a7e7705e95664dc669c